### ISSUE TITLE
Law Harvest/Hunt Issue

### ISSUE THREAD

**Issue by Sekor2600** [StrangeLoopGames/EcoIssues#18189]
![grafik](https://user-images.githubusercontent.com/37456949/92971746-55765b80-f481-11ea-8da4-580f3278df4b.png)
![grafik](https://user-images.githubusercontent.com/37456949/92971764-59a27900-f481-11ea-8ddc-a787631e5b9a.png)

You can't destroy the plants with the machete even though it is neither Harvest nor Hunt

**Comment by IcedForge** [comment]
Missing tool set for 
Machete
![Screenshot_39](https://user-images.githubusercontent.com/24981362/107381427-1cedb300-6aef-11eb-865d-b269e9d276ea.jpg)


Shovel when digging dirt
![Screenshot_37](https://user-images.githubusercontent.com/24981362/107381274-f92a6d00-6aee-11eb-92db-1b1a510493ca.jpg)
Shovel ( when harvesting a plant, such as camas bulb or taro root )
![Screenshot_38](https://user-images.githubusercontent.com/24981362/107381297-fdef2100-6aee-11eb-8f42-7bfc2aadf4d0.jpg)

**Comment by IcedForge** [comment]
#19076 is Separate issue but will but should be resolved by this fix, need to close it once this is done.

**Comment by tuha263** [comment]
After the discussion with @IcedForge and @Sekor2600, we will:
- Adding new Trigger `DestroyPlant`
- Using `DestroyPlant` instead of `HarvestAndHunt`  for Machete,...
- If shovel dig under plants, trigger `DestroyPlant' too.

**Comment by WeaselDog** [comment]
Pushing this
